autogen/main/outputs.tf.tmpl

```diff
@@ -146,6 +146,11 @@ output "service_account" {
   value       = local.service_account
 }
 
+output "netowrk" {
+  description = "The subnetwork to host the cluster in"
```

Comment:
I assume this should be network instead of subnetwork?

```suggestion
  description = "The cluster's network"
```

Reply:
I did one to many things when I created this PR.... It should actually be subnetwork.
I'm running the linting as well now to make sure that it passes. 
I will push the fix as soon as it's done.